views/js/pciCreator/ims/mathEntryInteraction/creator/tpl/alternativeForm.tpl

```diff
@@ -0,0 +1,15 @@
+<div class="math-entry-response-wrap">
+    <div class="math-entry-alternative-wrap" data-index="{{index}}">
+        <span class="math-entry-response-title math-entry-response-alternative">{{__ "Alternative"}}
+            <a href="#" class="answer-delete">
+                <span class="icon-bin"></span>
+                <span>{{__ "delete the answer"}}</span>
+            </a>
+        </span>
+        <span class="math-entry-input mq-math-mode mq-editable-field math-entry-alternative-input" style="display: block;">
```

Comment:
The goal is to have a specific CSS class that distinguishes the alternative input from the correct one.
I have kept the other CSS styles because the MathQuill uses and I want to avoid side effects if I remove those.

Reply:
Well, the question is not about the CSS classes, but regarding the `style="display: block;"`

Reply:
Ahh, I've checked, and it seems not to be needed at all.